qa/tasks/cephfs/test_volumes.py

```diff
@@ -2376,6 +2377,69 @@ def test_subvolume_clear_earmark(self):
         # get earmark
         get_earmark = self._fs_cmd("subvolume", "earmark", "get", self.volname, subvolume)
         self.assertEqual(get_earmark, "")
+    
+    def test_earmark_on_non_existing_subvolume(self):
+        subvolume = "non_existing_subvol"
+        earmark = "nfs.test"
+        with self.assertRaises(CommandFailedError):
```

Comment:
Expect the exact error ENOENT. Same for below cases.

Reply:
Done!